hearts/tests/test_game.py

```diff
@@ -108,3 +112,12 @@ def test_rankings():
                                     p1: 1,
                                     p2: 2,
                                     p3: 4}
+
+
+def test_deserialize_game_check_player_data():
+    test_players = [Player('Lauren'), Player('Erin'), Player('Jeremy'), Player('Daniel')]
+    test_game, _ = new_game(test_players, points_to_win=27)
+    test_game.players = test_players  # Seat players in the order: Lauren, Erin, Jeremy, Daniel
+    assert test_game.players == test_players
+    print ([player.username for player in test_game.players])
+    assert test_game.players == [Player('Lauren'), Player('Erin'), Player('Jeremy'), Player('Daniel')]
```

Comment:
@j2kun : I don't see why this fails.  Is it because when I created a new Game object `test_game` (and when it calls `shuffle(test_game.players)` when initialized), then `test_players` also gets randomized in place?